deployments/monitoring-env-production.nix

```diff
@@ -0,0 +1,10 @@
+{ ... }:
+{
+  monitoring = { ... }: 
```

Comment:
i believe it is equally valid to make the entire file just:
```
{
  monitoring = ../modules/production.nix;
}
```
but i also want to see `require` used more in iohk-ops, so it would be:
```
{
  require = [ ./monitoring.nix ];
  monitoring = ../modules/production.nix;
}
```
then `deployment/monitoring.nix` doesnt have to be passed in at `nixops modify` time, making the haskell side a bit simpler